ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -19,6 +19,9 @@
     state: present
     line: ::1 localhost.localdomain localhost
 
+- name: create NODE_TEST_DIR directory
+  file: path=/home/{{ server_user }}/tmp state=directory
```

Comment:
yes is created by root:
```
drwxr-xr-x  2 root root 4.0K May  5 17:48 tmp
```

I guess I should add also write permission as it will be used by jenkin to run test.